### ISSUE TITLE
repl it dont build the .env

### ISSUE THREAD

**Issue by bernienightscout** [malaow3/Glucose-Widget#5]
hi there, 

i want to build it your way, but replit change the way to add .env files. can you tell me a "how to" how i can add it? 9i use ios 15.1 is it possible to use this widget on my iphone? have you a new how to for it or can you create it?

**Comment by malaow3** [comment]
Just checked on replit and unfortunately they did change things... Kinda annoying on their end, but it's a simple enough fix. To add env variables now, go to the "Secrets" tab (the lock icon). Then the easiest way to do it is scroll down to the "Open raw editor" and click on that. The just copy/paste the following, with your details obviously:

`{
"username": "Enter your username here",
"password": "Enter your password here"
}`

Then just click save! Let me know if there are any issues w/ that!

**Comment by bernienightscout** [comment]
ok i will trie it. but which username and password should or i have to use?

and whats with key and value above?

**Comment by malaow3** [comment]
username and password are for you Dexcom account (e.g. the account info you use to access / share data with others in the app)

you can ignore the key / value stuff, since we set that using the raw json editor

**Comment by bernienightscout** [comment]
ok...

i run it but it exit with status 2. is tzhis ok? and where i can find the link of the webpage which is set up?

**Comment by malaow3** [comment]
status 2 indicates a failure; could you send a screenshot the output and attach here (making sure there are no leaked usernames / passwords, wouldn't want you to accidentally send those)

**Comment by bernienightscout** [comment]
Von meinem iPhone gesendet

> Am 27.10.2021 um 21:41 schrieb malaow3 ***@***.***>:
> 
> status 2 indicates a failure; could you send a screenshot the output and attach here (making sure there are no leaked usernames / passwords, wouldn't want you to accidentally send those)

**Comment by bernienightscout** [comment]
![Unbenannt](https://user-images.githubusercontent.com/79535762/139137367-74433a40-04eb-4e3a-9210-16268e5b2093.JPG)

**Comment by malaow3** [comment]
Ah seems like one of the libraries has updated and it has broken compatibility; I'd suggest maybe using the python setup in the meantime

**Comment by bernienightscout** [comment]
![Unbenannt](https://user-images.githubusercontent.com/79535762/139141324-a6e5e519-24c0-445b-892f-0a972947abf5.JPG)

**Comment by malaow3** [comment]
Hmmm that is odd that both aren't working. Are you a Dexcom user outside of the United States by chance? If that is the case, that could be the root of the issue as Dexcom has different endpoints for OUS users.

**Comment by bernienightscout** [comment]
im from germany but i have both accounts i could check it with my us account…



Von meinem iPhone gesendet

**Comment by malaow3** [comment]
Yes, please try with the US based account, and if that doesn't work I can look into how to help you set up the OUS account. Thanks!

**Comment by bernienightscout** [comment]
![Unbenannt](https://user-images.githubusercontent.com/79535762/139294866-5c4a9747-ecea-430d-9291-c1c7b840fa80.JPG)

**Comment by bernienightscout** [comment]
i think its the same

**Comment by malaow3** [comment]
Here is a precompiled binary of the Go code (make sure to unzip), you should be able to just drag+drop this into repl and then run with the following -- make sure you are on the shell tab!

`./getBG`

Let me know if there are still any issues - and try running w/ the US account first. 

[getBG.zip](https://github.com/malaow3/Glucose-Widget/files/7436461/getBG.zip)

**Comment by bernienightscout** [comment]
![Unbenannt](https://user-images.githubusercontent.com/79535762/139314225-cb020827-377e-483d-930a-c94c26f50538.JPG)

**Comment by malaow3** [comment]
Please rename the file to remove the "._" before hand. The file name should be just "getBG" and then run it with `./getBG` like you had just done!
